cmd/kubeadm/app/cmd/upgrade/diff.go

```diff
@@ -184,7 +184,9 @@ func runDiff(fs *pflag.FlagSet, flags *diffFlags, args []string, fetchInitConfig
 			Context:  cmdutil.ValueFromFlagsOrConfig(fs, "context-lines", upgradeCfg.Diff.DiffContextLines, flags.contextLines).(int),
 		}
 
-		difflib.WriteUnifiedDiff(flags.out, diff)
+		if err = difflib.WriteUnifiedDiff(flags.out, diff); err != nil {
+			return err
```

Comment:
```suggestion
			return fmt.Errorf("write unified diff: %w", err)
```

Reply:
Thanks, I fixed it. ❤️

Reply:
Yes, that's why I've merged these trivial code change.